lib/tasks/install-blueprint.js

```diff
@@ -10,7 +10,7 @@ const path = require('path');
 const merge = require('ember-cli-lodash-subset').merge;
 const execa = require('../utilities/execa');
 const SilentError = require('silent-error');
-const validateNpmPackageName = require('validate-npm-package-name');
+const npa = require('npm-package-arg');
```

Comment:
Did we already have this in the package.json (I see that `validate-npm-package-name` is removed, but don't see this added)?

Reply:
confirm